this patch fixes gcc warning:
format '%u' expects type 'unsigned int', but argument 5 has type 'size_t'

more info at: http://www.dinkumware.com/htm_cl/lib_prin.html

--- libsemanage-1.2/src/module_ps_api.c.orig	2005-08-25 18:04:22.000000000 +0200
+++ libsemanage-1.2/src/module_ps_api.c	2005-09-27 03:24:50.117674500 +0200
@@ -536,7 +536,7 @@
         va_start (ap, fmt);
         (void) vsnprintf (buf, sizeof (buf), fmt, ap);
         va_end (ap);
-        (void) snprintf (buf2, sizeof (buf2), "%s %u\n", buf, data_len);
+        (void) snprintf (buf2, sizeof (buf2), "%s %zd\n", buf, data_len);
         if (write (fd, buf2, strlen (buf2)) == -1) {
                 return -1;
         }
